web-service/README.md

```diff
@@ -1,40 +1,94 @@
+(read below for more details).
+
+This particular implementation is made to work with the sample Physical Web
+applications in this repo.
 
-This particular implementation is made to work with the sample Physical Web applications in this repo.
 
 ## Why do we use a Physical Web Service?
 
-UriBeacon devices have an open but very specific protocol which must be followed.  Anyone is free to do so.  This is the only requirement for participation and interaction with the Physical Web.
+UriBeacon devices have an open but very specific protocol which must be
+followed.  Anyone is free to do so.  This is the only requirement for
+participation and interaction with the Physical Web.
+
+Any app/device can read these UriBeacon packets and do with them whatever they
+wish.  We have provided some example apps (and published them to app stores) to
+make it easier for you to try out the Physical Web.
 
-Any app/device can read these UriBeacon packets and do with them whatever they wish.  We have provided some example apps (and published them to app stores) to make it easier for you to try out the Physical Web.
+Our app of course wants to provide a beautiful UX, with rich information about
+the URLs it finds, but even more so, it wants to protect the user and his/her
+privacy from the many potential misuses of the physical web.  We hope every
+other app will do so, also!
 
-Our app of course wants to provide a beautiful UX, with rich information about the URLs it finds, but even more so, it wants to protect the user and his/her privacy from the many potential misuses of the physical web.  We hope every other app will do so, also!
+One really nice property of the Physical Web is that UriBeacons cannot
+physically detect when clients scan it (and so cannot track passers-by).
+However, the web servers actually backing these URLs can of course track all
+requests made for them (as per usual on the web).  So, our app must be
+extremely careful with how it uses the URLs it finds.
 
-One really nice property of the Physical Web is that UriBeacons cannot physically detect when clients scan it (and so cannot track passers-by).  However, the web servers actually backing these URLs can of course track all requests made for them (as per usual on the web).  So, our app must be extremely careful with how it uses the URLs it finds.
+Yet, showing raw URLs alone is not descriptive enough for most users, and is
+basically useless to anyone when they are obfuscated strings coming from an URL
+shortener (which is common due to the URL length limit).
 
-Yet, showing raw URLs alone is not descriptive enough for most users, and is basically useless to anyone when they are obfuscated strings coming from an URL shortener (which is common due to the URL length limit).
+We would like to grab a nice Title / Icon from the actual content of the page
+-- but that requires making a request for the page -- which may potentially be
```

Comment:
New sentence starting at 'but'. Introduce clause starting at 'which' with a comma instead of '--'.